src/main/java/com/minecolonies/colony/Structures.java

```diff
@@ -72,14 +86,31 @@ private static void loadStyleMaps()
                 basePath = Paths.get(uri);
                 loadStyleMaps(basePath);
             }
+
+            File decorationFolder;
+
+            if(FMLCommonHandler.instance().getMinecraftServerInstance() == null)
+            {
+                 decorationFolder = new File(Minecraft.getMinecraft().mcDataDir, "minecolonies/decorations");
+            }
+            else
+            {
+                decorationFolder = new File(FMLCommonHandler.instance().getMinecraftServerInstance().getDataDirectory(), "minecolonies/decorations");
+            }
+
+            if(!decorationFolder.exists() && !decorationFolder.mkdirs())
+            {
+                Log.getLogger().warn("Failed to create directories for dynamic decorations.");
+            }
+            loadStyleMaps(decorationFolder.toPath());
         }
         catch (@NotNull IOException | URISyntaxException e)
         {
-            Log.getLogger().error("Error loading Schematic directory. Things will break!", e);
+            Log.getLogger().error("Error loading StructureProxy directory. Things will break!", e);
         }
     }
 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'hut' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)